ext/node/ops/worker_threads.rs

```diff
@@ -66,7 +66,7 @@ where
   match node_resolver.url_to_node_resolution(url)? {
     NodeResolution::Esm(u) => Ok(u.to_string()),
     NodeResolution::CommonJs(u) => wrap_cjs(u),
-    NodeResolution::BuiltIn(_) => Err(generic_error("Neither ESM nor CJS")),
+    _ => Err(generic_error("Neither ESM nor CJS")),
```

Comment:
Can you revert this to not use a wildcard pattern match? When we add or change variants on `NodeResolution` we should get a compiler error here.